### ISSUE TITLE
Windows symlink issues

### ISSUE THREAD

**Issue by lbogdan** [yarnpkg/berry#278]
Hello there,

Love the work you're doing with yarn v2, and can't wait to have an official release!

Until then, is the current `2.0.0` preview release supposed to work in a Windows 10 environment? I followed the steps [here](https://yarnpkg.github.io/berry/getting-started/install), but whenever I do `yarn install`, I get lots of
`➤ YN0001: │ Error: webpack-cli@npm:3.3.5: EPERM: operation not permitted, symlink '..\cache\webpack-cli-npm-3.3.5-b312d4a5784d744ee30d1030065733a9e4bfa30175a83a503d24293354e87b1a.zip' -> 'C:\Users\Bogdan\work\webpack\.yarn\virtual\webpack-cli-virtual-a20e759eb0a7c0e23d7e204a2d2137a56ea80da5a4be32033bcb060d9e0c65d5'`
errors.

Let me know if / how I can help in debugging this.

**Comment by arcanis** [comment]
Can you check whether the Developer Mode is enabled on your Windows? Cf here:

**Comment by lbogdan** [comment]
Wow, I've been using Windows for development for so long, but I had no idea there's such a thing as *"Developer Mode"*! 😃 That indeed fixed it! (and alternatively running it in an elevated shell, of course).

Now I'm left with two other issues:
- `webpack-cli` doesn't work
```
$ yarn webpack-cli
C:\Users\Bogdan\work\webpack\.pnp.js:11069
        throw firstError;
        ^

Error: ENOENT: no such file or directory, stat 'C:\Users\Bogdan\work\webpack\.yarn\virtual\webpack-cli-virtual-a20e759eb0a7c0e23d7e204a2d2137a56ea80da5a4be32033bcb060d9e0c65d5'
    at Object.statSync (fs.js:855:3)
[...]
```
- `local-web-server` doesn't work:
```
$ yarn ws
Error: Couldn't find a suitable Node resolution for the specified unqualified path

Source path: /C:/Users/Bogdan/work/webpack/lws-lws-basic-auth
Rejected resolution: /C:/Users/Bogdan/work/webpack/lws-lws-basic-auth
Rejected resolution: /C:/Users/Bogdan/work/webpack/lws-lws-basic-auth.js
Rejected resolution: /C:/Users/Bogdan/work/webpack/lws-lws-basic-auth.json
Rejected resolution: /C:/Users/Bogdan/work/webpack/lws-lws-basic-auth.node

    at Object.makeError (C:\Users\Bogdan\work\webpack\.pnp.js:9962:19)
[...]
```

Should I open separate issues, or can we debug them here?

**Comment by arcanis** [comment]
We can continue here (although it might be easier to do this on Discord, for more immediate interactions). Regarding:

- `webpack-cli`, I would check what is `webpack-cli-virtual-a20e75...`. It should be a symlink pointing to a file in `.yarn/cache` - if either the symlink or the target don't exist, there's a problem somewhere.

- `local-web-server`, I'm not sure what that is, but the error message explains that something is trying to run `/C:/Users/Bogdan/work/webpack/lws-lws-basic-auth`, but no matching script exist (hence why it enumerates the rejected extensions). A quick search on their GitHub repository reveals no reference to `lws-lws-basic-auth`, so I'm gonna assume it's something you define yourself. How is `local-web-server` configured to use it?

**Comment by arcanis** [comment]
We debugged it on Discord; fixes should be in #279 and #281 😃
